openedx/core/djangoapps/programs/tasks/v1/tests/test_tasks.py

```diff
@@ -145,7 +145,7 @@ def test_completion_check(
         programs.
         """
         tasks.award_program_certificates.delay(self.student.username).get()
-        mock_get_completed_programs.assert_called(self.site, self.student)
```

Comment:
Invalid assertion for mock 1.0.1, and checks the wrong call after upgrading (called for each site).  Fixed to make sure that a call was made with these arguments.